File: m2-h2/src/main/java/ru/yegorpilipenko/otus/spring/m2h2/Application.java
package ru.yegorpilipenko.otus.spring.m2h2;

import org.springframework.boot.autoconfigure.SpringBootApplication;

import javax.annotation.Nonnull;

import static org.springframework.boot.SpringApplication.run;

@SpringBootApplication
public class Application {

    public static void main(@Nonnull final String... args) {
        run(Application.class, args);
    }
}

File: m2-h1/src/main/kotlin/ru/yegorpilipenko/otus/spring/m2h1/Application.kt
package ru.yegorpilipenko.otus.spring.m2h1

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.transaction.annotation.EnableTransactionManagement

@SpringBootApplication
@EnableTransactionManagement
class Application

fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
